okapi-core/src/test/java/org/folio/okapi/bean/BeanTest.java

```diff
@@ -286,21 +238,13 @@ public void testModuleDescriptorTimers() {
       + "  } ]" + LS
       + "}";
 
-    try {
-      final ModuleDescriptor md = Json.decodeValue(docModuleDescriptor,
-        ModuleDescriptor.class);
-      String pretty = Json.encodePrettily(md);
-    } catch (DecodeException ex) {
-      ex.printStackTrace();
-      fail = 400;
-    }
-    assertEquals(0, fail);
+    ModuleDescriptor moduleDescriptor = Json.decodeValue(docModuleDescriptor, ModuleDescriptor.class);
+    moduleDescriptor.setPermissionSets(null);
+    assertNull(moduleDescriptor.getPermissionSets());
```

Comment:
Do not see the value of these two lines.

Reply:
Makes coverage 100%, rather than 91%